docs/building.md

```diff
@@ -133,19 +132,15 @@ Make sure to install the `C++ CMake tools for Windows` component for Visual Stud
    For the 64-bit version of the dependencies please run this command:
 
    ```
-   vcpkg install fmt:x64-windows sdl2:x64-windows sdl2-ttf:x64-windows libsodium:x64-windows gtest:x64-windows
+   vcpkg install fmt:x64-windows sdl2:x64-windows sdl2-ttf:x64-windows libsodium:x64-windows libpng:x64-windows gtest:x64-windows
    ```
 
    For the 32-bit version of the dependencies please run this command:
 
    ```
-   vcpkg install fmt:x86-windows sdl2:x86-windows sdl2-ttf:x86-windows libsodium:x86-windows gtest:x86-windows
+   vcpkg install fmt:x86-windows sdl2:x86-windows sdl2-ttf:x86-windows libsodium:x86-windows libpng:x86-windows gtest:x86-windows
    ```
 
-* **Manually**
-1. Download and place the MSVC Development Libraries of [SDL2](https://www.libsdl.org/download-2.0.php), [SDL2_ttf](https://www.libsdl.org/projects/SDL_ttf/) and [Libsodium](https://github.com/jedisct1/libsodium/releases) in `%USERPROFILE%\AppData\Local\Microsoft\WindowsApps\`.
-2. If dependencies are not found or you wish to place them in other location - configure required path variables in _"Manage Configurations..."_ dialog inside Visual Studio or in _cmake-gui_.
-
```

Comment:
I feel like that video really needs some audio to help frame each step of the process. Like mentioning what webpages you're visiting and what each of the steps are meant to accomplish.

Reply:
Still not very used to recording my voice, but I might give it a shot, net time I install a Windows VM :D

Do you think the video is better then nothing, or should we just keep it as text only until someone do somethting better?

Reply:
It's hard for me to say since I don't really use video tutorials myself. I feel like this video doesn't do much besides following the steps on the text guide so it may not be that useful for most. However, for anyone who has never used Visual Studio or really any IDE before, it might be useful to have this as a visual guide to verify that they're pressing the right buttons and getting the same feedback.

Reply:
Yeah visually verifying that they where pressing all the required buttons was the intent of the video. But I think I might be best to just have it handy when ever someone comes to the chat saying they couldn't get it to work, and then add a nicer once once it is available.